README.md

```diff
@@ -1,10 +1,11 @@
 <!--- Copyright (c) 2013 Gordon Williams, Pur3 Ltd. See the file LICENSE for copying permission. -->
-EspruinoDocs
-============
+# EspruinoDocs
```

Comment:
Yeah, they are the same - except *every other heading in the entire repo uses dashes*, so it feels a bit pointless to randomly change these to something different?